menyer/pizza_house/assets/pizza_house.css

```diff
@@ -0,0 +1,127 @@
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    width: 100%;
+    max-height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+#main {
+    max-height: 100%;
+    width: 100%;
+    display: flex;
```

Comment:
flex and grid were not part of the assignment but I did not specify this when you did the PR so you don't need to change it.